apps/tokenomics/components/Home/BondingProducts.jsx

```diff
@@ -50,24 +64,43 @@ export const BondingProducts = () => {
   return (
     <>
       <PageHeader isMobile={isMobile}>
-        <Title level={4} className="mb-0 mt-0">
-          Bonding Products
-        </Title>
-        <ResponsiveDivider />
-        <Radio.Group onChange={onChange} value={bondingProgramType}>
-          <Radio value={BONDING_PRODUCTS.ALL}>All</Radio>
-          <Radio value={BONDING_PRODUCTS.ACTIVE}>Active</Radio>
-          <Radio value={BONDING_PRODUCTS.INACTIVE}>Inactive</Radio>
-        </Radio.Group>
-        <ResponsiveDivider />
-        <SwitchContainer>
-          <Switch
-            checked={hideEmptyProducts}
-            onChange={onToggle}
-            className="mr-8"
-          />
-          Hide empty products
-        </SwitchContainer>
+        <Flex justify="space-between" align="center" style={{ width: '100%' }}>
```

Comment:
can we handle this inline style like we do in `apps/tokenomics/components/Home/Bonding/Deposit/Deposit.jsx` or by adding a custom class to global styles